docs/src/main/sphinx/functions/json.rst

```diff
@@ -17,6 +17,9 @@ Trino supports three functions for querying JSON data:
 is based on the same mechanism of exploring and processing JSON input using
 JSON path.
 
+Trino also supports two functions for generating JSON data:
```

Comment:
either make it a sentence without a : .. or keep the : but then actually enumerate them in a list. In this case I would probably just replace : with -